content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,87 @@
+## Installation
+
+- **Added** experimental [manifest and profile commands](docs/setup/install/operator/) to install and manage the Istio control plane for evaluation.
+
+## Traffic Management
+
+- **Added** automatic determination of protocol types. You no longer need to explicitly name service ports as `http` or `tcp`.
```

Comment:
Can you link to https://preliminary.istio.io/docs/ops/traffic-management/protocol-selection/

Reply:
accepted.